### ISSUE TITLE
trading_api.get_favorites() does not exist

### ISSUE THREAD

**Issue by jeroent321** [Chavithra/degiro-connector#150]
According to the docs, section 7.12, there is a 

`favorites_batch = trading_api.get_favorites()`

which should get you your favorite products. This does not seem to exist. `trading_api.get_favorite()` does exist, this gives you the favorites lists, their ids and names. How to get the products from such a list?

**Comment by Chavithra** [comment]
Hello there,

*FAVORITES*
It was `get_favorite` instead of `get_favorites`.

I have fixed it in the documentation.

*PRODUCTS*
Once you have the products you can use the `get_products_info` to get product details.

I hope that helps.

**Comment by jeroent321** [comment]
Well, the thing is that `get_favorite` doesn't actually return products. It returns the names and ids of your favorite lists. Once you have those, you should be able to request which products are in one of those favorite lists, and then you can use `get_products_info` to get info on each product. But the function to request the product(id)s in the list seems to be missing.

**Comment by Chavithra** [comment]
After some digging here is what I have found:
- The endpoint consumed by the method `get_favorite` indeed returns `productIds`.
- The reason `productIds` is not returned is because this field is missing inside `FavoriteItem` model.

So the only way to get the `productIds` was to use the option `raw=True` when calling the method `get_favorite`.

I have updated the models and published a new version of the library.

Thanks for your feedback
